java/src/test/java/com/saucelabs/simplesauce/SauceSessionTest.java

```diff
@@ -3,29 +3,35 @@
 import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
 import org.openqa.selenium.MutableCapabilities;
 
 import org.openqa.selenium.WebDriver;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.net.MalformedURLException;
 
 import static org.junit.Assert.*;
 import static org.mockito.Mockito.*;
 
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({System.class})
 public class SauceSessionTest {
     //TODO duplication in 3 classes, excluding DataCenterTest
     private SauceSession sauce;
-    private EnvironmentManager dummyEnvironmentManager;
     private SauceRemoteDriver dummyRemoteDriver;
     private SauceOptions options;
 
     @Before
     public void setUp() {
-        //TODO duplication in setup in BaseConfigurationTest. Can be moved out of here
-        //and combined into a single setup()
         dummyRemoteDriver = mock(SauceRemoteDriver.class);
-        dummyEnvironmentManager = mock(EnvironmentManager.class);
-        sauce = new SauceSession(dummyRemoteDriver, dummyEnvironmentManager);
-        when(dummyEnvironmentManager.getEnvironmentVariable("SAUCE_USERNAME")).thenReturn("test-name");
-        when(dummyEnvironmentManager.getEnvironmentVariable("SAUCE_ACCESS_KEY")).thenReturn("accessKey");
+        PowerMockito.mockStatic(System.class);
+        PowerMockito.when(System.getenv(Mockito.eq("SAUCE_USERNAME"))).thenReturn("test-name");
+        PowerMockito.when(System.getenv(Mockito.eq("SAUCE_ACCESS_KEY"))).thenReturn("accessKey");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Mockito.eq' due to existing static import 'org.mockito.Mockito.*'](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4730520)